openshift/openshift-virtualization.rst

```diff
@@ -432,17 +449,12 @@ The following example permits the A10 GPU device and A10-24Q vGPU device.
           - externalResourceProvider: true
             pciDeviceSelector: 10DE:2236
             resourceName: nvidia.com/GA102GL_A10
-          mediatedDevices: # Include for vGPU
-          - externalResourceProvider: true
-            mdevNameSelector: NVIDIA A10-24Q
-            resourceName: nvidia.com/NVIDIA_A10-24Q
```

Comment:
The yaml snippet here is incorrect. We need to keep the mediatedDevices section instea of the pciHostDevices section. In addition, as mentioned in another comment let's specify the `A10-12Q` profile for consistency.